### ISSUE TITLE
[DefenseTech] [Missile Interceptor Server] [Table Error]

### ISSUE THREAD

**Issue by Paralaxus** [Andrew2070/AuraTech#1]
Reminder to fix the table error in the Missile Interceptor Server.
So that the table siloData {} prints out information of each silo respective to each ID value.
Currently it prints out the respective Id's and #'s but copies 1 silo's info into all other slots.

Should look like:
| Silo #: |ID #: | Missile Type: | Targets: |
     1           1        Nuclear           0,0,0
     2          2        Shrapnel         1,1,1

**Comment by Paralaxus** [comment]
Refer to this for more information:
Alright so i got the information to be sent wirelessly in near perfect form.

However there are a few problems i don't know how to fix.

1. The Screen prints the appropriate Silo #'s and their respective ID #'s, but only uses 1 slave's information to fill both's other fields.

For example rather than saying this:
Silo #1 Id#1 Type: Nuclear
Silo #2 Id#2 Type: Conventional

It says:

http://i.imgur.com/HAM27gg.png

There are two different computers with two different missiles and two different targets.
It should print the information respective to each individual silo, rather than copy paste 1 silo's information into the fields of the others.

2. What the computer should return if no missile is present, or if it is an anti-ballistic missile it should abbreviate it to A.B.M.
I know this sounds easy for a lot of people to do as a general if/else combo, but i can't seem to know where to put either.
Basically the slave's will crash when they find out that there is no missile, i dont know how to make them return another value.

3. How do i make the Target key of the table print x,y,z values?, right now it only prints X.

New working codes:
Master: http://pastebin.com/zTJ6WFj7
Slave: http://pastebin.com/fZ5xeB99
